hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
+    private static final String OLD_VALUE = "oldValue";
+    private static final String NEW_VALUE = "newValue";
 
-    static HazelcastInstance client;
-    static HazelcastInstance server;
+    private static HazelcastInstance client;
+    private static HazelcastInstance server1;
+    private static HazelcastInstance server2;
+    private static HazelcastInstance server3;
+
+    private String randomMapName = randomString();
+    private IMap<String, String> randomStringMap = client.getMap(randomMapName);
 
```

Comment:
Each test runs in its own thread, just static fields should be shared. Otherwise this would not work as it does (tests are all green).

`new Thread(new TestRunner(method, notifier), method.getName()).start();`